client-js/app/elements/labrad-plot.ts

```diff
@@ -132,6 +142,13 @@ export class Plot extends polymer.Base {
     left: PLOT_LEFT_MARGIN
   };
 
+  private fitBounds = {
+    xMin: 0,
+    xMax: 0,
+    yMin: 0,
+    yMax: 0
```

Comment:
I would prefer to use out-of-band sentinel values (e.g. `NaN`) here, but not sure if that complicates things elsewhere. I guess a degenerate box with zero area is also a fairly clear indication not to show the fit rect.